### PR TITLE
chore(examples): update variables example to use action configs

### PR DESCRIPTION
**Pull request by Walther**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes https://github.com/garden-io/garden/issues/3765

**Special notes for your reviewer**:

I noticed in the backend `main.go` and `Dockerfile` the port `8080` is used, so I edited that into the ingress definition (instead of `80`) and added it to the frontend as well (`const backendServiceEndpoint = 'http://backend:8080/hello-backend'`).
Let me know if there is a better way to fix this - I tried e.g. `containerPort: 8080, localPort: 80` on the backend ingress, but was unsuccessful.

### CODE REVIEW

**Comment by vvagaytsev** [comment]
Looks good! 👍 We should also update the `README.md` file, it still uses the term "module".

File: examples/variables/README.md [hunk 1]
```diff
@@ -4,16 +4,16 @@ This variant of the basic [demo project](../demo-project/README.md) demonstrate
...
 
 We also show how you can alternate these variables by the environment you're running, by overriding the default value
 in the `local` environment. In this case, we only want _one_ replica of each service while developing locally, but
 default to three when deploying remotely.
 
 ## Using variables with templating
 
-Also in the [backend module configuration](./backend/garden.yml#L6-L14) two variables are used to store ports configuration for the service and a variable
-is additionally used to detect dev-mode being used. Later in the [service ports configuration](./backend/garden.yml#L19) the variables are dynamically used to enable the development port if dev mode is being used. 
+Also in the [backend action configuration](./backend/garden.yml#L12) two variables are used to store ports configuration for the service and a variable
```
Comment (vvagaytsev) [inline examples/variables/README.md:16]: Nit: "backend action" -> "backend deploy action"

Comment (Walther) [reply examples/variables/README.md:16]: good catch, thanks!

**Review by vvagaytsev — APPROVED** [review]
Great job, thank you! 👍

**Pull request merged**